tutorials/2d/2d_parallax.rst

```diff
@@ -0,0 +1,227 @@
+
+Introduction
+------------
+
+Parallax is an effect used to simulate depth by having textures move at different speeds relative to the camera. It's
+also something that even experienced users struggle to wrap their heads around. Godot provides the
+:ref:`Parallax2D<class_parallax2d>` node to make this easier for users. It can still be easy to get tripped up though,
+so this page provides in-depth descriptions of some properties and how to fix some common mistakes.
+
+Scroll scale
+------------
+
+The backbone of the parallax effect is the :ref:`scroll_scale <class_parallax2d_property_scroll_scale>` property.
+It works as a scroll-speed multiplier, allowing layers to move at a different speed than the camera for each axis set.
+A value of 1 makes the parallax node scroll at the same speed as the camera. If you want your image to look further away
+when scrolling, use a value lower than 1, with 0 bringing it to a complete stop. If you want something to appear closer
+to the camera, use a value higher than 1, making it scroll faster.
+
+.. image:: img/2d_parallax_size_viewport.webp
+
+The scene above is comprised of five layers. Some good :ref:`scroll_scale <class_parallax2d_property_scroll_scale>`
+values might be:
+
+- ``(0.7, 1)`` - Forest
+- ``(0.5, 1)`` - Hills
+- ``(0.3, 1)`` - Lower Clouds
+- ``(0.2, 1)`` - Higher Clouds
+- ``(0.1, 1)`` - Sky
+
+The video below displays how these values affect scrolling while in-game:
+
+.. video:: video/2d_parallax_scroll_scale.webm
+   :alt: A scene with five layers scrolling at different speeds
+   :autoplay:
+   :loop:
+   :muted:
+
+Infinite repeat
+---------------
+
+:ref:`Parallax2D<class_parallax2d>` provides a bonus effect that gives textures the illusion of repeating infinitely.
+:ref:`repeat_size<class_parallax2d_property_repeat_size>` tells the node to snap its position forward or back when the
+camera scrolls by the set value. This effect is achieved by adding a single repeat to all the child canvas items offset
+by the value. While the camera scrolls between the image and its repeat, it invisibly snaps back giving the appearance
+of a looping image.
+
+.. image:: img/2d_parallax_scroll.gif
+
+Being a delicate effect, it's easy for unfamiliar users to make mistakes with their setup. Let's go over the "how" and
+"why" of a few common problems users encounter.
+
+Poor sizing
+~~~~~~~~~~~
+
+The infinite repeat effect is easiest to work with when you have an image designed to repeat seamlessly and is the same
+size or larger than your viewport **before** setting the :ref:`repeat_size<class_parallax2d_property_repeat_size>`. If
+you aren't able to obtain assets that are designed for this task, there are some other things you can do to better
+prepare your image in regards to size.
+
+Here is an example of a texture that is too small for its viewport:
+
+.. image:: img/2d_parallax_size_bad.webp
+
+We can see that the viewport size is ``500 x 300`` but the texture is ``288 x 208``. If we set the
```

Comment:
```suggestion
We can see that the viewport size is 500×300 but the texture is 288×208. If we set the
```
Either no formatting, or alternatively just for the numbers. I think this is fine? Also unsure whether we should add "pixel", but this is probably clear enough as is.